dnf-behave-tests/features/comps-upgrade.feature

```diff
@@ -0,0 +1,231 @@
+Feature: Group and environment upgrade
+
+
+# comps-upgrade-1 repo:
+#   A-group:
+#     mandatory: A-mandatory
+#     default: A-default
+#     optional: A-optional
+#     conditional: A-conditional-true (if dummy), A-conditional-false (if nonexistent)
+#   AB-group:
+#     mandatory: A-mandatory
+#     default: A-default
+#     optional: A-optional
+#     conditional: A-conditional-true (if dummy), A-conditional-false (if nonexistent)
+#   testenvironment:
+#     grouplist: A-group
+
+# comps-upgrade-2 repo:
+#   A-group:
+#     mandatory: A-mandatory
+#     default: A-default
+#     optional: A-optional
+#     conditional: A-conditional-true (if dummy), A-conditional-false (if nonexistent)
+#   B-group:
+#     mandatory: B-mandatory
+#     default: B-default
+#     optional: B-optional
+#     conditional: B-conditional-true (if dummy), B-conditional-false (if nonexistent)
+#   AB-group:
+#     mandatory: B-mandatory
+#     default: B-default
+#     optional: B-optional
+#     conditional: B-conditional-true (if dummy), B-conditional-false (if nonexistent)
+#   testenvironment:
+#     grouplist: B-group
+
+
+Background: Enable comps-upgrade-1 nad install dummy
+  Given I use repository "comps-upgrade-1"
+    And I successfully execute dnf with args "install dummy"
+
+
+Scenario: Upgrade group when there are new package versions - upgrade packages
+  Given I successfully execute dnf with args "group install A-group"
+    And I use repository "comps-upgrade-2"
+   When I execute dnf with args "group upgrade A-group"
+   Then the exit code is 0
+    And Transaction is following
+        | Action        | Package                            |
+        | upgrade       | A-mandatory-0:2.0-1.x86_64         |
+        | upgrade       | A-default-0:2.0-1.x86_64           |
+        | upgrade       | A-conditional-true-0:2.0-1.x86_64  |
```

Comment:
This test is failing for me and I don't see versions 2.0 of the packages in the `comps-upgrade-2` repository, did you by chance forget to add them?

Reply:
Sorry, yes, I forgot to add them.

Reply:
Thanks, works now. Are you planning on addressing https://github.com/rpm-software-management/ci-dnf-stack/pull/885#discussion_r480979965? I don't mind much merging this the way it is, but having the comment accurate would be better...

Reply:
It's fixed now. I was thinking if the comment is useful for this feature file, but I can't tell, since I wrote the feature myself and it's still fresh in my mind. But in general, I find these comments useful, so I am keeping it.